config.mk.in

```diff
@@ -54,15 +54,17 @@ plugindir  = @bcf_plugindir@
 pluginpath = @bcf_pluginpath@
 PLUGIN_EXT = @PLUGIN_EXT@
 
-@Hsource@HTSDIR = @HTSDIR@
+HTSDIR = @HTSDIR@
```

Comment:
This change from the first commit can be reverted due to the rewrite in the second commit too I think. Everything else (apart from the _test/test-regidx_ rule fixup) is a Windows-only addition, but this (theoretically) affects the other platforms too.